docs/sources/tempo/setup/upgrade.md

```diff
@@ -15,6 +15,108 @@ This upgrade guide applies to on-premise installations and not for Grafana Cloud
 
 >**TIP**: You can check your configuration options using the [`status` API endpoint]({{< relref "../api_docs#status" >}}) in your Tempo installation.
 
+## Upgrade to Tempo 2.3
+
+Tempo 2.3 has several considerations for any upgrade:
+
+* vParquet3 is available as a stable, production-read block format
+* Configuration option to use Azure SDK v2
+* New `defaults` block in Overrides module configuration
+* Several configuration parameters have been renamed or removed.
+
+For a complete list of changes, enhancements, and bug fixes, refer to the [Tempo 2.3 changelog](https://github.com/grafana/tempo/releases).
+
+### Production-ready vParquet3 block format
+
+vParquet3 provides improved query performance and [dedicated attribute columns]({{< relref "../operations/dedicated_columns" >}}).
+
+This block format is required for using structural operators and dedicated attribute columns.
+
+While vParquet2 remains the default backend for Tempo 2.3, vParquet3 is available as a stable option. 
+Both work with Tempo 2.3.
+
+Upgrading to Tempo 2.3 doesn’t modify the Parquet block format. 
+
+{{% admonition type="note" %}}
+Tempo 2.2 can’t read data stored in vParquet3.
+{{% /admonition %}}
+
+Recommended update process:
+
+1. Upgrade your Tempo installation to version 2.3, remaining on vParquet3.
+2. Verify the upgrade is stable and performs as expected.
+3. [Change the block format to vParquet3]({{< relref "../configuration/parquet" >}}).
+
+If you notice any issues, you can immediately downgrade to Tempo 2.2. All your data remains readable.
```

Comment:
This statement seems to conflict with the note on line 41. Should this actually be downgrade to vParquet2 instead of Tempo 2.2?
```suggestion
If you notice any issues, you can immediately downgrade to vParquet2. All your data remains readable.
```

Reply:
If, on step 2 they notice issues they can downgrade to 2.2 and data will remain readable

If they notice issues on step 3 then they will have vp3 blocks and cannot downgrade to 2.2 w/o data loss

Reply:
Thank you. I've updated the information.